packages/grid/k8s/manifests/ingress.yaml

```diff
@@ -10,8 +10,13 @@ spec:
       name: proxy
       port:
         number: 80
+  tls:
+    - hosts:
+        - "localhost"
```

Comment:
@madhavajay, we do need to add this currently, it was fixed in a earlier PR that was merged

Reply:
an empty "" value is invalid, it needs to have something in there if the key exists. For live servers you should override this in the helm chart values, but since this is being used in k3d locally it needs a value

Reply:
Oh, I see you removed it from the `dev` branch code, so it acts as a wildcard again.

Reply:
We add tls and hostname based routing while we create helm charts, but we omit for local devspace deployments.